updates/2_add_nodejs_env.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+WORKING_DIRECTORY="/home/dev"
+
+die() {
+  echo "$*" >&2
+  exit 444
+}
+
+success() {
+  echo "$*" >&2
+  exit 0
+}
+
+main() {
+  pacman -S nodejs npm
```

Comment:
You should add --no-confirm tag. The user already accept rhe installation on the update system.
Update it like this:
'''pacman -S nodeja npm --no-confirm

Reply:
noconfirm option added.